aiidalab_widgets_base/viewers.py

```diff
@@ -168,44 +168,46 @@ class _StructureDataBaseViewer(ipw.VBox):
     DEFAULT_SELECTION_RADIUS = 6
     DEFAULT_SELECTION_COLOR = "green"
 
-    def __init__(self, configure_view=True, **kwargs):
+    def __init__(
+        self,
+        configure_view=True,
+        configuration_tabs=["Selection", "Appearance", "Cell", "Download"],
+        default_camera="orthographic",
+        show_camera_selector=True,
```

Comment:
@danielhollas I was wondering, would it make sense for your use case to move the camera setting into another tab (along with "Selection", "Appearance")? Then we could manage its appearance just like everything else.

Reply:
That seems fine. Actually, wouldn't it make sense if the camera toggle was part of the appearance tab? It seems a bit wasteful to have an extra tab just for this.

Reply:
> Actually, wouldn't it make sense if the camera toggle was part of the appearance tab? 

fine with me.